JASP-Desktop/widgets/listmodelinteractionassigned.cpp

```diff
@@ -40,6 +39,23 @@ void ListModelInteractionAssigned::initTerms(const Terms &terms, const RowContro
 	ListModelAssignedInterface::initTerms(interactionTerms(), allOptionsMap);
 }
 
+const Terms &ListModelInteractionAssigned::terms(const QString &what) const
+{
+	if (what == "noInteraction")
+	{
+		static Terms terms;
+
+		terms.clear();
```

Comment:
Why not do: `terms = { _fixedFactors, _randomFactors, _covariates };` ?

Reply:
Terms does not support this construction

Reply:
Aah, wouldn't it be nice to add that? Well, I guess it isn't so important because we will remove/change Terms anyway right?